pkg/client/unversioned/request.go

```diff
@@ -585,8 +592,20 @@ func (r *Request) Watch() (watch.Interface, error) {
 	if client == nil {
 		client = http.DefaultClient
 	}
+	if r.backoffMgr == nil {
+		panic("nil backoffMgr should never occur")
```

Comment:
Not needed. Should never happen, and the very next line will produce a nil pointer panic anyway.